### DISCUSSION TITLE
New Source: IMAP Server

### DISCUSSION THREAD

**Discussion by ilpizze** [airbytehq/airbyte#35261] [New Connector Request]
## Tell us about the new integration you’d like to have
Source: IMAP Server

## Describe the context around this new integration
Sync all emails on the server with a database table

## Describe the alternative you are considering or using
Self made batch in java, python or Pentaho PDI

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368155784203) by [Unito](https://www.unito.io)

**Comment by sherifnada** [comment]
@ilpizze that's a really interesting use case. I see there are clients like https://github.com/mjs/imapclient which can help make implemenation easier. What do you imagine the interface to this connector would look like? what inputs would a user supply, and what should the output look like? 

Also, do you have a particular use case in mind that would benefit from a connector like this? Can you help us understand how a business or individual might derive value from this connector?

**Comment by ilpizze** [comment]
The input should me an email folder. The output should be a database table. 

In my company we do a lot of nlp, nlu and machine learning in general on mails and documents for our clients. Actually for ETL/ELT we are using a mix of Pentaho PDI, Python code and Java Quartz Jobs denending on out clients. It would be really nice to have a unified (and modern) tool for all the ETL/ELT.

[a comment by an automated account is not shown]
